content/articles/2018-10-ggplot2-3-1-0.Rmarkdown

```diff
@@ -0,0 +1,159 @@
+---
+title: 'ggplot2 3.1.0'
+author: Mara Averick and Claus Wilke
+date: '2018-10-26'
+slug: ggplot2-3-1-0
+description: > 
+  ggplot2 3.1.0 is now on CRAN!
+categories:
+  - packages
+photo:
+  url: https://unsplash.com/photos/Ibq4B5iE_-4
+  author: Stephan Henning
+---
+
+```{r setup, include = FALSE}
+library(ggplot2)
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+We're happy to announce the release of
```

Comment:
Should include a brief description of ggplot2 (I mean people _probably_ know what it is, but best to be sure)